app/controllers/AdminloginController.php

```diff
@@ -3,6 +3,31 @@
 
 class AdminloginController extends Base
 {
+    private static function getHashedPassword($password)
+    {
+        return sha1($password);
+    }
+
+    private static function isCorrectPassword($exceptedPassword, $givenPassword)
```

Comment:
This looks a bit much no? Creating one line methods that are called only once?

Reply:
SHA1 is not considered to be safe anymore.
https://en.wikipedia.org/wiki/SHA-1#Attacks
https://linuxfr.org/users/gouttegd/journaux/et-paf-le-sha-1

So it would be useful to change that in the future. If it is done, this method would be a bit longer: it would have to check the new hashing way and the old one (with SHA1) for compatibility.